### ISSUE TITLE
Engine fails to install in CentOS 8

### ISSUE THREAD

**Issue by sandoche2k** [zonemaster/zonemaster-engine#680]
```
t/Test-dnssec-more.t ...... ok

#   Failed test 'afnic.fr gives ALL_ALGO_SIGNED'
#   at t/Test-dnssec.t line 19.

#   Failed test 'afnic.fr does not give RRSIG_BROKEN'
#   at t/Test-dnssec.t line 30.
# Looks like you failed 2 tests of 189.
t/Test-dnssec.t ...........
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/189 subtests
        (less 2 skipped subtests: 185 okay)
t/Test-dnssec05-A.t ....... ok
```

**Comment by sandoche2k** [comment]
Vincent  will you merge this issue since Mats will be back next week only. I will test and get back to you.

**Comment by vlevigneron** [comment]
Review it, I will merge it.

le 26 déc., sandoche2k a ?crit :
> Vincent  will you merge this issue since Mats will be back next week only. I will test and get back to you.
> 
> -- 
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub:
> https://github.com/zonemaster/zonemaster-engine/issues/680#issuecomment-569021106

-- 
	Vincent Levigneron  A.F.N.I.C.  Vincent.Levigneron@afnic.fr

**Comment by matsduf** [comment]
Resolved by #681
